### ISSUE TITLE
ReceiveDemo_Advance - Compiler error IDE 1.6.6

### ISSUE THREAD

**Issue by bdonaubauer** [sui77/rc-switch#56]
Hello Board,

after compiling the Advance Demo i got the following compiler error

exit status 1
'char* dec2binWzerofill(long unsigned int, unsigned int)' was declared 'extern' and later 'static' [-fpermissive]

I have tried the actual IDE version 1.6.11 and the older version 1.6.2.

Thanks for helping

**Comment by bau-sec** [comment]
Does anyone know the status of the Travis CI builds?  The build badge in the readme links to a page saying this project isn't active. There have been a couple compile errors come up as issues recently (this one and #51) which CI builds could be catching.

**Comment by fingolfin** [comment]
@bau-sec Hmm, seems to be broken. To be honest, I never was even aware it existed in the first place... Aha, I see, it was merged as part of @38 by @sui77 but I don't know whether he ever proceeded to actually activate it on the Travis side... ?

@sui77 I think you will have to become active to fix the Travis problem. At least I don't think I have the permissions to deal with it. I just tried, but Travis told me `There was an error while trying to activate the repository`. My guess is that I cannot "activate" things as I don't own this repository. Which makes sense, too.
